test/layers/stateless.jl

```diff
@@ -12,6 +12,20 @@ const ϵ = 1e-7
   @testset "mse" begin
     @test mse(ŷ, y) ≈ (.1^2 + .9^2)/2
   end
+  
+  @testset "mae" begin
+    @test Flux.mae(ŷ, y) ≈ 1/2
+  end
+  
+  @testset "huber_loss" begin
+    @test Flux.huber_loss(ŷ, y) ≈ 0.0012499999999999994
```

Comment:
With my suggestion from above:

```suggestion
    @test Flux.huber_loss(ŷ, y) ≈ 0.20500000000000002
```